<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AesKeyGenerator</class>
 <widget class="QDialog" name="AesKeyGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AES Key Generator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTextEdit" name="textEdit">
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblLength">
       <property name="text">
        <string>l√§nge:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="cmdGenerate">
       <property name="text">
        <string>generate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="cmdSetAsKey">
       <property name="text">
        <string>set AES key</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="chkSoZe">
       <property name="text">
        <string>Sonderzeichen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
